rero_ils/modules/documents/jsonschemas/documents/document_contribution_organisation-v0.0.1.json

```diff
@@ -63,7 +63,6 @@
       "title": "Conference",
       "description": "Mark as conference if the access point is directly at the conference name and not at the corporate body name",
       "type": "boolean",
-      "default": false,
```

Comment:
Currently, this field is present in all contribution of type "corporate body". The easiest way is to let the default value and make the field required.
If we removing the default value, we need to do mass corrections.